src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/PluginOverlay.kt

```diff
@@ -0,0 +1,40 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import javafx.geometry.Pos
+import javafx.scene.Node
+import javafx.scene.layout.Pane
+import javafx.scene.layout.Priority
+import javafx.scene.layout.StackPane
+import javafx.scene.paint.Color
+import tornadofx.*
+
+class PluginOverlay: StackPane() {
```

Comment:
Maybe call this ProgressOverlay? Then we could reuse for the import RC and create a project progress displays